packages/mapper/src/mapper.integration.spec.ts

```diff
@@ -86,7 +86,7 @@ describe('Mapper (integration)', function() {
 
       describe('when calling it after find', () => {
         it('returns next batch of docs', async() => {
-          mapper.find(collection, {}, { _id: 0 });
+          mapper.collectionMapper.find(collection, {}, { _id: 0 });
```

Comment:
This is a TODO. I have to convert this tests to separate files and just have `collectionMapper.find`